src/test/resources/v2_notifications_response.json

```diff
@@ -11,7 +11,6 @@
       "line_4": "a line 4",
       "line_5": "a line 5",
       "line_6": "a line 6",
-      "line_7": "a line 7",
```

Comment:
Why was this removed?

Reply:
I removed it because we don't actually return that line as far as I am aware.
While testing different client-libraries, I kept seeing line_7 but I did not see the mention on the response and I did some digging and it seems that we do not return line_7 [here](https://github.com/alphagov/notifications-api/blob/7730ed50a32f43f8d119009c23b7f0d8eb6c3360/app/models.py#L1612)